src/cpp/transport/TCPTransportInterface.cpp

```diff
@@ -1087,13 +1092,15 @@ void TCPTransportInterface::SocketAccepted(TCPAcceptor* acceptor, const asio::er
         }
         else
         {
-            logError(RTPC, "Incomming connection from unknown Acceptor: " << IPLocator::getPhysicalPort(acceptor->mLocator));
+            logError(RTPC, "Incomming connection from unknown Acceptor: "
+                << IPLocator::getPhysicalPort(acceptor->mLocator));
             return;
         }
     }
     else
     {
         logInfo(RTCP, " Accepting connection failed (error: " << error.message() << ")");
+        eClock::my_sleep(200); // Wait a little to accept again.
```

Comment:
Could you explain the reason for this sleep?

Reply:
Sometimes, the acceptor was being stuck on error accepting incoming connections until I included the sleep. It is possible that the error wasn't related to the wait, but I think it doesn't hurt.